src/Utils/Helpers.spec.ts

```diff
@@ -81,29 +85,31 @@ describe('Helpers', () => {
       // act
       const result = Helpers.getClusterUrls(args);
       //assert
-      result.api.should.be.equal('https://test.com');
-      result.bus.should.be.equal('amqps://rabbit.test.com');
+      expect(result).toEqual({
+        api: 'https://test.com',
+        bus: 'amqps://rabbit.test.com'
+      });
     });
   });
 
   describe('getExecArgs', () => {
     it('should return current exec args', () => {
       const { args, command } = Helpers.getExecArgs();
-      command.should.eq(process.execPath);
-      args.should.have.members([...process.execArgv, ...process.argv.slice(1)]);
+      expect(command).toBe(process.execPath);
```

Comment:
I'm not sure, but if follow our concept, we have to test a hole object.

Reply:
Good catch, replaced with `toMatchObject` bc `getExecArgs()` returns more than args and commands